.buildkite/hooks/post-checkout

```diff
@@ -32,6 +37,27 @@ checkout_merge() {
     }
 }
 
+# Usage:
+#check_if_file_exist_in_repo "infra" "main" "cd/release/release-manager/project-configs" "build.gradle"
+#Returns IS_FILE_EXISTS_IN_REPO=true if file exists and IS_FILE_EXISTS_IN_REPO=false if not exists
+check_if_file_exist_in_repo() {
+    local repoName=${1}
+    local branch=${2}
+    local fileName="build.gradle"
+
+    if [[ $branch == "main" ]]; then
+        local branchName="master"
+    fi
+
+    local path_to_file="cd/release/release-manager/project-configs/${branchName}/${fileName}"
```

Comment:
If `branch` is not main, `branchName variable is not created.

What value should have that `branchName` variable if `branch` is not `main`?

Reply:
fixed